### PR TITLE
Modify students,experts,volunteers section in home page

### LINKED ISSUE #325: Modify Students/Experts section
**Is your feature request related to a problem? Please describe.**
The current implementation is a bit different than the original design. (Students/Experts section in index.html page)

**Describe the solution you'd like**
- Please make it similar to the provided design. (Should be similar to the provided design, colors, borders, font sizes, etc..)
- Set text alignment to justify.
- All the custom CSS rules should add to the `global-custom.css` file.

Design: 
Screenshot 2020-01-09 at 18 25 09

### PR DESCRIPTION
**Pull request by yohanym95**
## Purpose

The purpose of this PR is to

## Goals

Modify Students/Experts section

## Approach

Removed unnecessary paragraph sections and add them into single paragraph.
Apply CSS style into paragraph. 

### Screenshots

  
s1
s2
s3

### Preview Link

https://pr-386-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
Windows 10, Google Chrome

## Learning

N/A

### CODE REVIEW

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
It should be similar to the provided design, colors, borders, Font sizes etc..

**Comment by yohanym95** [comment]
@jayasanka-sack  how is it about now?

**Comment by jayasanka-sack** [comment]
> Should be similar to the provided design, colors, borders, Font sizes etc..
@yohanym95 

Reffer to the provided design. It is in purple color. And the sizes, borders, font colors are different from the implementation

**Comment by yohanym95** [comment]
@jayasanka-sack how is it about now?

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
Great Job! @yohanym95 
Set text alignment on the card to justify

File: src/main/webapp/assets/css/global-custom.css [hunk 1]
```diff
@@ -446,3 +446,15 @@ body{
 {
     background-color: rgb(243, 151, 120);
 }
+
+/* tab section */
+a.nav-link.mb-sm-3.mb-md-0.py-3{
+    color : #000000 ;
+    background-color:#FFFFFF;
+}
+a.nav-link.mb-sm-3.mb-md-0.py-3.active{
```
Comment (jayasanka-sack) [inline src/main/webapp/assets/css/global-custom.css:455]: The selector names look a little bit messy.
@Gravewalker666, Please suggest good selectors & suggest a way to optimize these rules.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -140,19 +140,19 @@ <h2 class="font-weight-900 text-dark text-roboto text-center text-uppercase">Our
                             <li class="nav-item">
                                 <a class="nav-link mb-sm-3 mb-md-0 active py-3" id="tabs-text-1-tab" data-toggle="tab"
                                    href="#tabs-text-1" role="tab" aria-controls="tabs-text-1" aria-selected="true">
-                                    <span class="display-4 py-2 text-uppercase text-roboto">students</span>
+                                    <span class="display-4 py-2 px-3 text-uppercase text-roboto" style="font-size: 20px;">students</span>
```
Comment (jayasanka-sack) [inline src/main/webapp/index.html:143]: Avoid using inline CSS rules

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/index.html (hunk 2 above)
Comment (yohanym95) [reply src/main/webapp/index.html:143]: i'll fix it.

**Comment by Gravewalker666** [comment]
@yohanym95 Thank you!
You have done all the hard work to complete the issue. There are only some minor changes we need to do to complete it fully. Since we have to publish the site by morrow I will merge this and get on with it.
Cheers mate!

**Pull request merged**